app-connector-client/server/app.js

```diff
@@ -67,6 +68,17 @@ function configValidation(configJson) {
         for (var i = 1; i <= events.length; i++) {
             {
                 var event = events[i - 1];
+                var specInJson
```

Comment:
please move it behind filename validation

Reply:
Done